metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -37,6 +37,12 @@
         <display-name>Metron apps indexed HDFS dir</display-name>
     </property>
     <property>
```

Comment:
Should these HDSF directories more specifically pertain to topology processing?  As the other use cases came into being, just 'error' may be too generic